sdk/cmake/host-tools.cmake

```diff
@@ -15,10 +15,41 @@ function(setup_host_tools)
         set(HOST_EXE_SUFFIX ".exe")
     endif()
 
+    if(CMAKE_SYSTEM_PROCESSOR EQUAL CMAKE_HOST_SYSTEM_PROCESSOR)
+        set(HOST_TOOLS_CMAKE_COMMAND "${CMAKE_COMMAND}")
+        message("Not cross-compiling, no special host-tools cmake command")
+    elseif(MSVC)
+        message("Compiling on ${CMAKE_HOST_SYSTEM_PROCESSOR} for ${CMAKE_SYSTEM_PROCESSOR} (MSVC)")
+        set(HOST_TOOLS_CMAKE_COMMAND "${REACTOS_BINARY_DIR}/host-tools/cmake_shim.cmd")
```

Comment:
Where does that file come from?

Reply:
if you continue reading for 2 lines, you have your answer.